libindy/src/api/blob_storage.rs

```diff
@@ -0,0 +1,38 @@
+extern crate libc;
+
+use api::ErrorCode;
+use errors::ToErrorCode;
+use commands::{Command, CommandExecutor};
+use commands::blob_storage::BlobStorageCommand;
+use utils::cstring::CStringUtils;
+
+use self::libc::c_char;
+
+#[no_mangle]
+pub extern fn indy_open_reader(command_handle: i32,
```

Comment:
May be rename to indy_open_blob_reader ???

Reply:
or `indy_blob_storage_open_reader` ?